imports/api/posts/methods.js

```diff
@@ -21,5 +22,12 @@ Meteor.methods({
             throw new Meteor.Error('you are not the owner and cant delete that post');
         }
         PostService.remove(_id);
+    },
+    'post.lists'(isApproved) {
+        console.log(isApproved);
```

Comment:
remove this

Reply:
we don't have security here. We shouldn't allow normal users to view posts that are not approved, See specs for more details